readthedocs/restapi/templates/restapi/footer.html

```diff
@@ -1,3 +1,8 @@
+{% load cache %}
+
+{# Vary the cache on the version #}
+{% cache 60 footer-html version.pk %}
```

Comment:
What about adding `versions.count` as a key also here and increasing the time of the cache a little more?

Reply:
I actually realized this isn't specific to the version, but to the project as a whole, so that should be even better.

Reply:
The VCS block actually will change by page, so I need to think through this a bit more.